functional-tests/src/test/groovy/pages/admin/external/LoginPage.groovy

```diff
@@ -0,0 +1,18 @@
+package Admin.pages
+
+import geb.Page
+
+class LoginPage extends Page {
+    // todo update/sanitize if possible
+
+    static url = "https://sso.pathfinder.gov.bc.ca/auth/realms/eagle/protocol/openid-connect/auth?client_id=eagle-admin-console&response_mode=fragment&response_type=code&scope=openid&redirect_uri=http%3A%2F%2Flocalhost%3A4200%2Fadmin"
```

Comment:
Localhost may need to be updated

Reply:
Yeah, the redirect should go to our dev deployment. This will get tested and updated in the first test suite I write.